test/user.test.js

```diff
@@ -16,7 +16,6 @@ describe("Create forum user successfully", function() {
             .post('/api/v1/users')
             .send({
                 username: 'Bob123',
-                displayName: 'bob',
```

Comment:
It think it could be better to have both a passing test with a given displayName and one without

Reply:
Sounds good to me, done.